packages/global/components/blocks/content-load-more.marko

```diff
@@ -39,10 +41,10 @@ $ const {
   <marko-web-load-more
     header=`More in ${sectionName}`
     component-name="content-load-more-flow"
-    component-input={ aliases, nativeX: input.nativeX }
+    component-input={ primarySectionAlias, aliases, nativeX: input.nativeX }
     fragment-name="content-list"
     query-name="website-scheduled-content"
-    query-params={ sectionId, excludeContentIds: [id], limit: 14, skip: sectionSkip }
+    query-params={ sectionId, excludeContentIds: [id], limit: limit, skip: sectionSkip }
```

Comment:
@Shinsina  I would ensure that `limit` is set in all three cases in this file.  And just update the company and Contact limit to match standard content for consistency.